yarn-project/p2p/src/service/data_store.test.ts

```diff
@@ -17,7 +17,7 @@ import { fromString as uint8ArrayFromString } from 'uint8arrays/from-string';
 
 import { AztecDatastore } from './data_store.js';
 
-const CLEANUP_TIMEOUT = 60_000;
+const CLEANUP_TIMEOUT = 80_000;
```

Comment:
```suggestion
const CLEANUP_TIMEOUT = 120_000;
```

If we're gonna bump, let's bump properly